modules/eco-self-node-remediation-operator-troubleshooting.adoc

```diff
@@ -36,12 +36,22 @@ If the `MachineHealthCheck` controller did not create the `SelfNodeRemediation`
 If the `SelfNodeRemediation` CR was created, ensure that its name matches the unhealthy node or the machine object.
 
 [id="daemon-set-exists_{context}"]
-== Daemon set exists even after uninstalling the Self Node Remediation Operator
-Issue:: The self node remediation daemon set exists even after after uninstalling the Operator.
+== Daemon set and other Self Node Remediation Operator resources exist even after uninstalling the Operator
+Issue:: The Self Node Remediation Operator resources, such as the daemon set, configuration CR, and the remediation template CR, exist even after after uninstalling the Operator.
 
-Resolution:: To remove the self node remediation daemon set, manually delete the `SelfNodeRemediationConfig` CR. Run the following command:
+Resolution:: To remove the Self Node Remediation Operator resources, manually delete the resources by running the following commands for each resource type:
```

Comment:
Delete "manually." Since you're giving the commands, "manually" doesn't add anything.

Reply:
Makes sense. Updated now